hpx/lcos/promise.hpp

```diff
@@ -139,20 +133,19 @@ namespace lcos {
         //   - promise_already_satisfied if its shared state already has a
         //     stored value or exception.
         //   - no_state if *this has no shared state.
-        void set_exception(
-            boost::exception_ptr const& e, error_code& ec = throws)
-        {
-            base_type::set_exception(e, ec);
-        }
+        using base_type::set_exception;
```

Comment:
On Donnerstag, 23. Juni 2016 13:30:15 CEST Hartmut Kaiser wrote:
> > @@ -139,20 +133,19 @@ namespace lcos {
> > 
> >          //   - promise_already_satisfied if its shared state already has
> >          a
> >          //     stored value or exception.
> >          //   - no_state if *this has no shared state.
> > 
> > -        void set_exception(
> > -            boost::exception_ptr const& e, error_code& ec = throws)
> > -        {
> > -            base_type::set_exception(e, ec);
> > -        }
> > +        using base_type::set_exception;
> 
> Do we generate docs using doxygen for this? I didn't think so.

Just checked, we don't, but we probably should, and those are already almost 
doxygen comments.
> 
> ---
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/STEllAR-GROUP/hpx/pull/2227/files/64f79a91506d10b1b247f82
> a5a9305bb72113870#r68308989